pkg/keyspace/keyspace.go

```diff
@@ -555,3 +571,66 @@ func (manager *Manager) allocID() (uint32, error) {
 	}
 	return id32, nil
 }
+
+// PatrolKeyspaceAssignment is used to patrol all keyspaces and assign them to the keyspace groups.
+func (manager *Manager) PatrolKeyspaceAssignment() error {
+	return manager.store.RunInTxn(manager.ctx, func(txn kv.Txn) error {
```

Comment:
do we need to add txn in other operators?

Reply:
What do you mean?